lessons/beginners/testing/index.md

```diff
@@ -133,14 +133,19 @@ test_secteni.py::test_secti ␛[32mPASSED␛[0m
 ␛[32m============= 1 passed in 0.00 seconds =============␛[0m
 ```
 
-Tento příkaz projde všechny soubory v aktuálním
-adresáři, jejichž jméno začíná na `test_`, zavolá v nich všechny funkce,
+Tento příkaz projde zadaný soubor, zavolá v něm všechny funkce,
 jejichž jméno začíná na `test_`, a ověří, že nevyvolají žádnou
-výjimku — typicky výjimku z příkazu `assert`.
+výjimku – typicky výjimku z příkazu `assert`.
```

Comment:
Dobře změněno, ale kompletně irelevantní pro tuto PR :D

Reply:
Přijde mi zbytečné kvůli tomuhle zakládat další PR. Ideální by bylo to na to mít extra commit, ale i to je spíš zbytečná byrokracie.

Reply:
taky jsme to jen zmínil (protože to nemám rád), ale nic jsem nenakazoval (jako změň to, apod.)

Reply:
Jasný. Ale nejspíš to tak vyznělo :)
Každopádně díky za review.

Reply:
příště to explicitně řeknu, že je to jen moje OCD :hospital:

Reply:
Já zas tak často takhle veřejně nepřispívám, tak mi přišlo hezké si udělat revert, čistě pro pocit „tahle to má být“.